CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+CHANGES IN VERSION 1.6.0 beta 1 (from 1.5.3)
+===================================
+
+This release of Appium marks the beginning of support for two brand new platforms:
+* [Windows](https://www.microsoft.com/en-us/windows) desktop applications
+* [You.i TV](http://www.youi.tv/) (see [driver documentation](https://github.com/YOU-i-Labs/appium-youiengine-driver#appium-youi-engine-driver)).
+
+Further, since Apple removed the Instruments automation functionality in Xcode 8,
+this release of Appium has preliminary support for XCUITest, allowing for the automation
+of applications in iOS 9.3 and 10.
+
+#### General
+- Require Node 4 or above
+- Add `automationName` entries for `XCUITest`, and `youiengine`
+- Add `platformName` entry for `Windows`
+
+#### iOS
+- Add preliminary support for Xcode 8 and iOS 10 (using `automationName` of `XCUITest`)
```

Comment:
should link to the doc where we include any additional setup instructions like carthage etc, otherwise people will be constantly pinging us about why it's not just working